config/shared/errors/errors.go

```diff
@@ -67,6 +67,8 @@ var (
 	ErrPartitionsUnitsMismatch     = errors.New("cannot mix MBs and sectors within a disk")
 	ErrSizeDeprecated              = errors.New("size is deprecated; use sizeMB instead")
 	ErrStartDeprecated             = errors.New("start is deprecated; use startMB instead")
+	ErrNeedLabelOrNumber           = errors.New("partition number or label must be specified")
```

Comment:
We probably want to be more specific here, since specifying 0 explicitly also triggers this. Maybe add "non-zero" in front?